package df.base.internal.confio;

public class ConfioException extends Error {

    public ConfioException(String message) {
        super(message);
    }

    public ConfioException(String message, Throwable cause) {
        super(message, cause);
    }

}
